src/js/stories/PostCard.stories.js

```diff
@@ -104,6 +158,7 @@ stories.add('Default', () => (
       ]}
       isDraft={boolean('Draft', false)}
       isInvalid={boolean('Invalid', false)}
+      isNative={boolean('Invalid', true)}
```

Comment:
This should be called "Is Native" so it doesn't conflict with the "Invalid" boolean control:

```
isNative={boolean('Is Native', true)}
```